deploy/tekton/toolchain.openshift.dev_v1alpha1_tektoninstallation_cr.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: toolchain.openshift.dev/v1alpha1
```

Comment:
I would not have separate dir for `tekton` and `che` rather I have one dir (something like `resources` / `crs` / or something used elsewhere in other community operator) and put both resource yaml file in one dir.

Reply:
as per https://github.com/codeready-toolchain/toolchain-operator/pull/16#discussion_r350581980 I will create a single dir with the same name as in `pkg` so we have a direct mapping between the YAML assets and their siblings embedded in go code.

Reply:
@nurali-techie please see b26c03e in which I restored the `pkg/che` and `pkg/tekton` packages and in which the go-bindata assets are generated (for Tekton only for now, Che will come next). So we have a direct mapping between `deploy/tekton` and `pkg/tekton`.
Also, I moved the code from `pkg/resources` to `pkg/toolchain`.

Reply:
@xcoulon thanks for refactoring.  Overall, I was thinking to have one dir to hold all cr_yaml file and one dir to hold all generated assets files.  Having assets file in different dir, we need to have clean target to clean individual file .. but now I would like to stop here and put that on you completely.

Reply:
I'm curious - why do we need this file at all? The spec is empty, so why couldn't we create the resource programmatically in go? Is it because of a better understanding of what is being created or is there something else behind it?
On the other hand, I guess that the Che installation file will be maybe more complicated - in that case it make sense for such a file... Then it would also make sense to do it for TektonInstallation file to keep consistency

As for the dirs - yes, I would prefer one directory for both files - I have a feeling that we overuse the directories and packages (mainly in this project). As for the name - why not `installation` or `installation-cr`, but to be clear - I don't insist on it ;-)